File: Assignment 4/doctor-soa/src/main/java/ds/health/model/PatientEntity.java
package ds.health.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "patient")
public class PatientEntity {

    @Id
    @Column(name = "id")
    private Integer id;

    @Column(name = "recommendation")
    private String recommendation;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getRecommendation() {
        return recommendation;
    }

    public void setRecommendation(String recommendation) {
        this.recommendation = recommendation;
    }
}

File: Assignment 4/caregiver-soa/src/main/kotlin/ds/health/caregiversoa/model/PatientEntity.kt
package ds.health.caregiversoa.model

import javax.persistence.*

@Entity
@Table(name = "patient")
class PatientEntity(
        @Id
        @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "patient_sequence_generator")
        @SequenceGenerator(name = "patient_sequence_generator", sequenceName = "patient_sequence", allocationSize = 1)
        var id: Int? = null,

        @Column(name = "recommendation")
        var recommendation: String? = null
)